sdk/keyvault/azure-keyvault-secrets/src/main/java/com/azure/security/keyvault/secrets/SecretAsyncClient.java

```diff
@@ -304,22 +306,22 @@
      * <p><strong>Code Samples</strong></p>
      * <p>Gets latest version of the secret, changes its notBefore time and then updates it in the Azure Key Vault.
      * Subscribes to the call asynchronously and prints out the returned secret details when a response is received.</p>
-     * {@codesnippet com.azure.keyvault.secrets.secretclient.updateSecret#secretBase}
+     * {@codesnippet com.azure.keyvault.secrets.secretclient.updateSecretProperties#secretProperties}
      *
-     * <p>The {@code secret} is required and its fields {@link SecretBase#getName() name} and {@link SecretBase#getVersion()
+     * <p>The {@code secret} is required and its fields {@link SecretProperties#getName() name} and {@link SecretProperties#getVersion()
      * version} cannot be null.</p>
      *
-     * @param secret The {@link SecretBase base secret} object with updated properties.
-     * @return A {@link Mono} containing the {@link SecretBase updated secret}.
+     * @param secret The {@link SecretProperties base secret} object with updated properties.
+     * @return A {@link Mono} containing the {@link SecretProperties updated secret}.
      * @throws NullPointerException if {@code secret} is {@code null}.
-     * @throws ResourceNotFoundException when a secret with {@link SecretBase#getName() name} and {@link
-     *     SecretBase#getVersion() version} doesn't exist in the key vault.
-     * @throws HttpRequestException if {@link SecretBase#getName()}  name} or {@link SecretBase#getVersion() version} is
+     * @throws ResourceNotFoundException when a secret with {@link SecretProperties#getName() name} and {@link
+     *     SecretProperties#getVersion() version} doesn't exist in the key vault.
+     * @throws HttpRequestException if {@link SecretProperties#getName()}  name} or {@link SecretProperties#getVersion() version} is
      *     empty string.
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
-    public Mono<SecretBase> updateSecret(SecretBase secret) {
-        return updateSecretWithResponse(secret).flatMap(FluxUtil::toMono);
+    public Mono<SecretProperties> updateSecretProperties(SecretProperties secret) {
```

Comment:
In some places, we are calling this property `SecretProperties secretProperties` and at some `SecretProperties secret`.
Should keep this consistent and call them secretProperties` throughout..

Reply:
fixed everywhere